Assets/Creatures/api/PlayableCreatureBase.cs

```diff
@@ -141,5 +153,17 @@ public virtual void HandleMovement(Vector2 inputDirection)
             Animator.SetFloat("DirectionX", inputDirection.x);
             Animator.SetFloat("DirectionY", inputDirection.y);
         }
+        private void OnTriggerEnter2D(Collider2D collision)
+        {
+            // Receive damage
+            // TODO: Based on target
+            playerHealth -= 10;
+            if (playerHealth == 0)
+            {
+                print("YOU DIED!");
```

Comment:
lmao

Reply:
should this be `Debug.Log`

Reply:
Yeah, I copy pasta'd your health script stuff to keep it working, but I think we'll need to revisit this code anyway